services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/StepIntersectionTest.java

```diff
@@ -38,12 +38,12 @@ public void testSerializable() throws Exception {
 
   @Test
   public void testToFromJson1() {
-    StepIntersection intersection
-      = StepIntersection.builder()
+    StepIntersection intersection = StepIntersection.builder()
       .out(0)
       .entry(Arrays.asList(true))
       .bearings(Arrays.asList(125))
       .rawLocation(new double[]{13.426579, 52.508068})
+      .tunnelName("test tunnel")
```

Comment:
This isn't the best test case, because there are no `classes` field, maybe add a new test case?

Reply:
@LukasPaczos right now `tunnelName` and `classes` are not connected. There are no checks if `classes` contains `tunnel`. I think it's a backend responsibility to set `tunnelName` only for `tunnel` class, we just deserialize it.
wdyt? should we handle it on our side?

Reply:
No that's fine. This was more of a nitpick :)